client/state/post-types/schema.js

```diff
@@ -0,0 +1,23 @@
+export const items = {
+	type: 'object',
+	additionalProperties: false,
+	patternProperties: {
+		'^\\d+$': {
+			type: 'object',
+			additionalProperties: {
+				type: 'object',
+				required: [ 'name' ],
+				properties: {
+					name: { type: 'string' },
+					label: { type: 'string' },
+					labels: { type: 'object' },
+					description: { type: 'string' },
+					map_meta_cap: { type: 'boolean' },
+					capabilities: { type: 'object' },
```

Comment:
>Also optional, I'm pretty sure our capabilities look like:

I like this recommendation, but I'll probably forgo [your earlier comment](https://github.com/Automattic/wp-calypso/pull/3251/files#r52644950). I think there's a balance to be struck here between being extreme strictness and ease of implementation/readability.